25-minimize.ks

```diff
@@ -18,8 +23,10 @@ systemctl disable sshd.service
 
 # This seems to cause 'reboot' resulting in a shutdown on certain platforms
 # See https://tickets.puppetlabs.com/browse/RAZOR-100
-echo " * remove intel mei modules"
-rm -rf /lib/modules/*/kernel/drivers/misc/mei
+# ^^ URL above doesn't exists anymore.
```

Comment:
https://github.com/puppetlabs-toy-chest/razor-el-mk/blob/77eeb878116cae5ad6165b482b30328dbcbf58aa/microkernel.ks#L155-L164 is where it still shows up. Note that is a kickstart for EL7 so I'd hope that has been resolved with EL9.